### DISCUSSION TITLE
2 ArcGIS Server Sites for Different Roles

### DISCUSSION THREAD

**Discussion by mikiekelly** [Esri/arcgis-powershell-dsc#533] [Q&A] [answered]
If I want to create 2 separate server sites (each with 2 nodes), one GIS Server site and one Image Server site (both federated), do I need to invoke 2 separate config files? It seems like the ServerRole applies to all Server nodes, but just want to double check in case I'm misinterpreting things.

**Comment by cameronkroeker** [comment, marked as answer]
Hi @mikiekelly,

Each node that is specified within a single json configuration file will be joined to the same site (the config-store/directories will need to be a file share). Therefore, if you want separate sites then you will need a json configuration file for each AGS site.

Thanks,
Cameron K.

**Reply by mikiekelly** [reply]
Great thanks for quick reply @cameronkroeker - much appreciated.
